### PR TITLE
Document endpoint peer mapping

### LINKED ISSUE #657: Document ota.endpoint.peer.service.mapping configuration option

### PR DESCRIPTION
**Pull request by anuraaga**
Fixes #657

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,15 @@ is a representation of the entity producing telemetry.
 |--------------------------|--------------------------|------------------------------------------------------------------------------|
 | otel.resource.attributes | OTEL_RESOURCE_ATTRIBUTES | Used to specify resource attributes in format: key1=val1,key2=val2,key3=val3 |
 
+#### Peer service name
+
+The [peer service name](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/span-general.md#general-remote-service-attributes) is
+the name of a remote service that is being connected to. It roughly corresponds to `service.name` in the [Resource](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service) for the local service.
```
Comment (iNikem) [inline README.md:128]: why `roughly` and not `exactly`? You want to correlate them, don't you?

File: README.md [hunk 2]
```diff
@@ -122,6 +122,15 @@ is a representation of the entity producing telemetry.
...
+
+| System property                     | Environment variable              | Purpose                                                                      |
+|------------------------------------|------------------------------------|------------------------------------------------------------------------------|
+| otel.endpoint.peer.service.mapping | OTEL_ENDPOINT_PEER_SERVICE_MAPPING | Used to specify a mapping from hostnames or IP addresses to peer services, as a comma separated list of host=name pairs. The peer service name will be added as an attribute to a span whose host or IP match the mapping. For example, 1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api |
```
Comment (iNikem) [inline README.md:132]: Should you say attribute key here as well?

File: README.md (hunk 1 above)
Comment (anuraaga) [reply README.md]: Good point not sure why I took a conservative approach there

File: README.md (hunk 2 above)
Comment (anuraaga) [reply README.md]: I'm not sure I understood this suggestion but tried something, let me know if it's what you were looking for.

**Pull request merged**